doc_source/fargate-profile.md

```diff
@@ -6,7 +6,7 @@ The Fargate profile allows an administrator to declare which pods run on Fargate
 
 If a pod matches multiple Fargate profiles, Amazon EKS picks one of the matches at random\. In this case, you can specify which profile a pod should use by adding the following Kubernetes label to the pod specification: `eks.amazonaws.com/fargate-profile: <profile_name>`\. However, the pod must still match a selector in that profile in order to be scheduled onto Fargate\.
 
-When you create a Fargate profile, you must specify a pod execution role for the pods that run on Fargate using the profile\. This role is added to the cluster's Kubernetes [Role Based Access Control](https://kubernetes.io/docs/admin/authorization/rbac/) \(RBAC\) for authorization so that the `kubelet` that is running on the Fargate infrastructure can register with your Amazon EKS cluster and appear in your cluster as a node\. The pod execution role also provides IAM permissions to the Fargate infrastructure to allow read access to Amazon ECR image repositories\. For more information, see [Pod execution role](pod-execution-role.md)\.
+When you create a Fargate profile, you must specify a pod execution role for the kubelet that runs on Fargate pods using the profile\. This role is added to the cluster's Kubernetes [Role Based Access Control](https://kubernetes.io/docs/admin/authorization/rbac/) \(RBAC\) for authorization so that the `kubelet` that is running on the Fargate infrastructure can register with your Amazon EKS cluster and appear in your cluster as a node\. The pod execution role also provides IAM permissions to the Fargate infrastructure to allow read access to Amazon ECR image repositories\. For more information, see [Pod execution role](pod-execution-role.md)\.
```

Comment:
same comment as above, if we can keep this generic (or keep updating this so customers know exactly what procs are assuming this role) that would be awesome.

Right now it is Kubelet, but soon it will be both Kubelet and fluent-bit